src/components/Missions.js

```diff
@@ -0,0 +1,36 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { joinMission, leaveMission } from '../redux/missions/mission';
+
+const Missions = () => {
+  const missions = useSelector((state) => state.missionReducer);
+  const dispatch = useDispatch();
+  const missionTable = missions.map(({
+    id, name, description, reserved,
+  }) => (
+    <tr key={id}>
+      <td>{name}</td>
+      <td>{description}</td>
+      <td className="text-center"><span className={(!reserved) ? 'status-field unavailable-status ' : 'status-field available-status'}>{(!reserved) ? 'NOT A MEMBER' : 'ACTIVE MEMBER'}</span></td>
+      <td className="text-center"><button onClick={() => dispatch((!reserved) ? joinMission(id) : leaveMission(id))} className={(!reserved) ? 'button-mission join-mission' : 'button-mission leave-mission'} type="button">{(!reserved) ? 'JOIN MISSION' : 'LEAVE MISSION'}</button></td>
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/156244653-5599d59b-03ea-4689-aeb5-e6565b48134c.png)

Therefore:

- [ ] Please, kindly use the React conditional rendering syntax, instead of using ternary operators.

## Hints

You could build if-else statements using this syntax by using the `!` (NOT) operator like this: `{!mission.reserved && (what to do if the previous thing is NOT true)}` and still meet the requirement 💪